src/services/signup-services/list-additional-data-info-service.js

```diff
@@ -12,18 +11,27 @@ export const listAdditionalDataInfoService = async () => {
   return parseHttpResponse(httpResponse)
 }
 
+const adapt = (response) => {
+  response = [
+    ...response.slice(0, 3),
+    { id: 10, key: 'Full Name', required: true, show: false },
+    ...response.slice(3)
+  ]
+
+  return response
+}
```

Comment:
Sugiro colocar algo assim:

```suggestion
const adapt = (response) => {
   const positionFullName = 3;
   const fullName = { id: 10, key: 'Full Name', required: true, show: false };
   response.splice(indice, 0, fullName);

  return response
}
```

Reply:
Valeu pela sugestão. Nem tinha pensado nisso